cmd/main.go

```diff
@@ -188,3 +191,7 @@ func getProjectVersion() (bool, string) {
 	}
 	return true, projectInfo.Version
 }
+
+func printV1DeprecationWarning() {
+	fmt.Printf("warning: [deprecation] v1 projects are deprecated and will not be supported beyond Feb 1, 2020. Refer to https://book.kubebuilder.io/migration/guide.html for more details.\n")
```

Comment:
WDYT about: 
```suggestion
	fmt.Printf("warning: [deprecation] v1 projects are deprecated and will not be supported beyond Feb 1, 2020. See the https://book.kubebuilder.io/migration/guide.html for know how to migrate the project to from v1 to v2.\n")
```

Reply:
@camilamacedo86 Is the "See ... **for know** how to ..." correct English? Sorry for this question, I'm not a native speaker, thus I never heard this idiom :)

Reply:
@camilamacedo86 ping, I would be thankful if you clarify this moment for me.

Reply:
`... for know how to ...` and `project to from` are definetely typos. I would go for `... Refer to https://.. for instructions on how to migrate to v2.\n`. Saying from `v1` doesn't give that much info as you already said that `v1` is the one being deprecated.

Just my 2 cents.

Reply:
@Adirio agreed, thanks for help.

Reply:
I was not clear in my suggestion. Sorry. It is updated following.